### PR TITLE
Request revision to design of readme and added ways to help HKers and past achievements

### PR DESCRIPTION
**Pull request by MoistCode**
[Preview README](https://github.com/MoistCode/bandinchina)

- Added ways to help HKers based off of this [thread](https://www.reddit.com/r/HongKong/comments/cv0ws4/how_can_you_help_hong_kong_protests_from_abroad/?utm_source=share&utm_medium=ios_app&utm_name=iossmf).

- Added some images to make it a little more fun and inviting

- Added links that direct to subsections of the README.md

- Cleaned up links

- Merged the blacklist and whitelisted companies

- Removed whitelist.md

- Added images used in the README.md to the public assets folder

Edit: Added preview link

### CODE REVIEW

File: READMEv2.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  </h2>  
+</a>
+
+* Create gifs on my desktop because my laptop sucks
+* Diagonal blasts
+* Boss fights 
+* Ability to counter elemental bullets by changing player element
+* Power-ups
+* Different game music options
+* Scoring system
```
Comment (Medallyon) [inline READMEv2.md]: What

Comment (Medallyon) [reply READMEv2.md]: I just realised that's the first commit, not the latest.

Comment (MoistCode) [reply READMEv2.md]: Yeah I took it from a previous README format.

**Comment by Faunsce** [comment]
This is a stunning overhaul  @MoistCode ! i 100% agree we should pull this one in @caffeine-overload . Amazing work!

**Comment by MoistCode** [comment]
For those who are like me and too lazy to pull this to take a look, feel free to visit my forked repo at
https://github.com/MoistCode/bandinchina to preview my PR.

**Comment by yolstatrisch** [comment]
PR looks okay, great job! You might want to resize the flag of EU and PRC to keep it consistent with the rest of the flags.

**Comment by MoistCode** [comment]
> PR looks okay, great job! You might want to resize the flag of EU and PRC to keep it consistent with the rest of the flags.

You're right @yolstatrisch. I'll try to get that tonight and resolve any conflicts as they come along as well.

**Comment by caffeine-overload** [comment]
Looks pretty awesome

I merged in an addition to the whitelist which has caused a conflict. 

Just some thoughts, perhaps it would be better to keep the lists in their own files, and then the writeup and lists are put together in the github webpage? I feel like this would make adding to the lists easier, especially to people who are not programmers but still want to make an addition.

**Comment by MoistCode** [comment]
> 
> 
> Looks pretty awesome
> 
> I merged in an addition to the whitelist which has caused a conflict.
> 
> Just some thoughts, perhaps it would be better to keep the lists in their own files, and then the writeup and lists are put together in the github webpage? I feel like this would make adding to the lists easier, especially to people who are not programmers but still want to make an addition.

@caffeine-overload That makes sense. The reason I went with the route of having everything in one go is to have a list of content that can just immediately snaps to the information wanted but I think your way is much more manageable. 

I can turn that content list to some sort of navigation and break the different major sections into their own files.

**Comment by caffeine-overload** [comment]
Yeah, let's try that. Then the whole thing comes together on the github webpage

**Comment by caffeine-overload** [comment]
@MoistCode Just checking in because you haven't been active here for a while, how is it going?

**Comment by MoistCode** [comment]
@caffeine-overload Hey! Apologies for not checking in in a while. I haven't been able to really do much due to some personal stuff going on. I'll try to get this in in a few days.

**Comment by MoistCode** [comment]
@caffeine-overload Okay so this is quite a change and I had to mess around with some things to make multi pages work. Below are a list of changes I've made:

1. Changed the usage of `BEGIN:TABLE` and `END:TABLE` to `RENDER START` and `RENDER END`. This regex is definitely useful but the stuff we want to render may not always be tables. I've also removed some restrictions in the regex so now all you would need are the flags `<!-- RENDER START -->` and `<!-- RENDER END -->` to signifiy where to start rendering and where to end. This makes it easier to make content show up in the site versus the repo README.

2. Made the site a little more mobile friendly but the tables are still a pain in the butt that needs to be looked at for mobile.

3. You can now view the home, blacklist, and whitelist on both the repo and on the site.

4. Added code to test links for both dev and production environment. May need to revisit if we make the origin URL something else.

5. Added a simple pixel for state management

**Pull request merged**